.github/workflows/packaging.yml

```diff
@@ -51,3 +51,24 @@ jobs:
 
       - name: Create packages
         run: ./packpack/packpack
+
+      - name: Deploy packages
+        # We need this step to run only on push with tag.
+        if: ${{ github.event_name == 'push' && github.event.ref_type == 'tag' }}
```

Comment:
`ref_type` present only in  `github.event_name == 'create'`  https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types

Reply:
I changed the condition to `startsWith(github.ref, 'refs/tags/')`.

Reply:
It seems to work fine.